File: DataSummitSolution/DataSummitUI/src/app/profileAttributes/models/paperSize.ts
import { Drawing } from "../../drawings/models/drawing";
import { ProfileAttribute } from "./profileAttribute";

export class PaperSize {

	number: number;
	Name: string;
	PixelWidth: number;
	PixelHeight: number;
	PhysicalWidth?: number;
	PhysicalHeight?: number;
	Drawings: Drawing[];
	ProfileAttributes: ProfileAttribute[];

	constructor(number?: number, name?: string, pixelWidth?: number,
		pixelHeight?: number, physicalWidth?: number, physicalHeight?: number,
		drawings?: Drawing[], profileAttributes?: ProfileAttribute[])
	{
		this.number = number || 0;
		this.Name = name || "";
		this.PixelWidth = pixelWidth || 0;
		this.PixelHeight = pixelHeight || 0;
		this.PhysicalWidth = physicalWidth || 0 || null;
		this.PhysicalHeight = physicalHeight || 0 || null;
		this.Drawings = drawings;
		this.ProfileAttributes = profileAttributes;
	}
}

File: DataSummitSolution/DataSummitModels/DB/PaperSize.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class PaperSize
    {
        public PaperSize()
        {
            Documents = new HashSet<Document>();
            TemplateAttributes = new HashSet<TemplateAttribute>();
        }

        public byte PaperSizeId { get; set; }
        public string Name { get; set; }
        public int PixelWidth { get; set; }
        public int PixelHeight { get; set; }
        public decimal? PhysicalWidth { get; set; }
        public decimal? PhysicalHeight { get; set; }

        public virtual ICollection<Document> Documents { get; set; }
        public virtual ICollection<TemplateAttribute> TemplateAttributes { get; set; }
    }
}
